source/unified-test-format/tests/invalid/clientEncryption-wrong_client_type.yml

```diff
@@ -0,0 +1,18 @@
+description: clientEncryption-wrong_client_type
```

Comment:
I don't think this test applies anymore, as `client` was removed from `clientEncryption`. It's likely only still invalid because of `additionalProperties: false` on `clientEncryption`.